Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -75,12 +66,34 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
             Assert.False(isExist);
         }
 
-        public void AddTests()
+        /// <summary>
+        /// Gets Settings of Test selected by Id
+        /// </summary>
+        [Fact]
+        public async Task GetSettingsById()
+        {
+            var test = CreateTest("AOT 669");
+            await _testRepository.CreateTestAsync(test);
+            Assert.NotNull(test);
+            var testSettings = await _testRepository.GetTestSettingsAsync(test.Id);
+            var testName = testSettings.TestName;
+            Assert.Equal(testName, "AOT 669");
+        }
+
+        /// <summary>
+        /// Updates settings of a particular test with the help of Id
+        /// </summary>
+        [Fact]
+        public async Task UpdateTestSettingsById()
         {
-            _trappistDbContext.Test.Add(new DomainModel.Models.Test.Test() { TestName = "BBIT 123" });
-            _trappistDbContext.Test.Add(new DomainModel.Models.Test.Test() { TestName = "MCKV 123" });
-            _trappistDbContext.Test.Add(new DomainModel.Models.Test.Test() { TestName = "CU 123" });
-            _trappistDbContext.SaveChanges();
+            var test = CreateTest("AOT 669");
+            await _testRepository.CreateTestAsync(test);
+            var settingsToUpdate = await _testRepository.GetTestSettingsAsync(test.Id);
+            settingsToUpdate.TestName = "IIT BANGALORE";
+            await _testRepository.UpdateTestSettingsAsync(settingsToUpdate);
+            var TestName = "IIT BANGALORE";
+            await _trappistDbContext.SaveChangesAsync();
```

Comment:
what is the reason to write _trappistDbContext.SaveChangesAsync(); its already happening in UpdateTestSettingsAsync method right ?